scapy/contrib/automotive/xcp/scanner.py

```diff
@@ -34,19 +35,21 @@ def __init__(self, can_socket, id_range=None,
         :param sniff_time: time the scan waits for a response
                            after sending a request
         """
+        if add_padding:
```

Comment:
What if I call the XCPScanner the first time with `add_padding=True` and afterwards with `add_padding=False`?
I think the padding would still be applied for the second runner.

Probably better to use:
```python
conf.contribs["XCP"]["add_padding_for_can"] = add_padding
```